WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1439,6 +1439,44 @@ TaskEditHeaderTableCell = padding:0 @TaskViewWrapperPadding@;
 TaskEditHeaderRowLabel  = @TaskEditViewFontSize@; @Label@;
 TaskEditHeaderTextInput = @TaskEditViewFontSize@; @FixBoxModel@; @TransparentBg@; @FullWidth@;
 
+# Single task view style
+TaskViewGutterSpace     = @MailViewGutterSpace@
+TaskViewWrapperPadding  = @ContactViewWrapperPadding@
+TaskView                = @ContactView@;
+TaskViewBottomPane      = @ContactView@; height:auto; margin:@MailViewGutterSpace@ 0 0 0;
+TaskWrapper             = @ContactWrapper@;
+TaskHeader              = @ContactHeader@;
+TaskHeaderRow           = @ContactHeaderRow@;
+TaskHeaderCell          = @ContactHeaderCell@;
+TaskHeaderMain          = @ContactHeaderMain@;
+TaskHeaderImageWrapper  = @ContactHeaderImageWrapper@;
+TaskHeaderImage         = @ContactHeaderImage@; @TransparentBg@;
+TaskHeaderInfo          = @ContactHeaderInfo@;
+TaskHeaderTitle         = @ContactHeaderTitle@;
+TaskHeaderTags          = @ContactHeaderTags@;
+TaskHeaderMeta          = vertical-align:middle;
+TaskHeaderMetaTable     = @ContactHeaderLocationTable@;
+TaskHeaderMetaTableIcon = @ContactHeaderLocationIcon@;
+TaskHeaderMetaTableValue= @ContactHeaderLocationName@;
+TaskContent             = @ContactContent@; padding-bottom:0; 
+TaskRowWrapper          = @ContactRowWrapper@;
+TaskRowLabel            = @ContactRowLabel@;
+TaskRowValue            = @ContactRowValueWrapper@; @ContactRowValue@; display:block;
+TaskBody                = padding:0 @TaskViewGutterSpace@ @TaskViewWrapperPadding@ @TaskViewGutterSpace@;
+TaskInfo                = @ContactInfo@;
+
+# Task List view
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
```

Comment:
This is block of properties that are added as part of https://github.com/Zimbra/zm-web-client/pull/61

I have just moved (while resolving conflicts) it up so that it is logically related with other properties.